rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/CompactView.java

```diff
@@ -114,46 +99,60 @@ Canvas build() {
         return root;
     }
 
-    public void setViewMyNodes(boolean value) {
-        this.onlyMyNodes = value;
-    }
+    void setViewMyNodes(boolean value) {
+        if (value) {
+            root.removeMember(normalFlowCanvas);
+            root.addMember(myNodesFlowCanvas);
+        } else {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace the type specification in this constructor call with the diamond operator ("<>"). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2293)